optax/contrib/_schedule_free_test.py

```diff
@@ -164,5 +164,16 @@ def run(opt):
     params_wrapper = run(opt_wrapper)
     chex.assert_trees_all_close(params_shortcut, params_wrapper)
 
+  @parameterized.parameters(*_OPTIMIZERS_UNDER_TEST)
+  def test_scalar_preservance(self, opt_name, opt_kwargs):
```

Comment:
the name `test_scalar_preservance` is not very descriptive. Please either use a more precise name for the test or add a comment below explaining what behavior the function is testing